src/engine/renderer/tr_local.h

```diff
@@ -508,6 +508,21 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 		vertexHash_t *next;
 	};
 
+	enum
+	{
```

Comment:
Why `B_`?

Reply:
For bind, was lacking imagination.

Reply:
can also use `BIND_`

Reply:
The enum is only defined in gl_shader.cpp to number texture units so maybe it could be defined locally there?

Reply:
okok

Reply:
Hmm, no, it's used in both `tr_shade.cpp` and `gl_shader.cpp` (the first one sets, the second one reads).